conda/meta.yaml

```diff
@@ -17,7 +17,7 @@ requirements:
     - tbb-devel
   run:
     - appdirs
-    - numpy >=1.15.3
+    - numpy
```

Comment:
I don't think we can change this, we still need at least this version. There where some changes to the buffer handling we were relying on.

Reply:
okay, I can leave it in, I just thought 1.15 is so old now that it seemed odd to document this particular minimum required version when we don't for the other dependencies.

Reply:
I suppose the example of the Mantid dependency shows that we are still at risk getting really old versions?